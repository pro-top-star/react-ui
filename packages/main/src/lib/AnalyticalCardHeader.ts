import { AnalyticalCardHeader } from '../components/AnalyticalCard/header/AnalyticalCardHeader';

export { AnalyticalCardHeader };
