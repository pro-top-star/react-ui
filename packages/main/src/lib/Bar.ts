import { Bar } from '../components/Bar';

export { Bar };
