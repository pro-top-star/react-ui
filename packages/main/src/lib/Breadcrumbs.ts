import { Breadcrumbs } from '../components/Breadcrumbs';

export { Breadcrumbs };
