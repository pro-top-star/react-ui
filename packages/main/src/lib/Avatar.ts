import { Avatar } from '../webComponents/Avatar';

export { Avatar };
