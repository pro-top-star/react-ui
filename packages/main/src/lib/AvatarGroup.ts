import { AvatarGroup } from '../webComponents/AvatarGroup';
import type { AvatarGroupPropTypes } from '../webComponents/AvatarGroup';

export { AvatarGroup };
export type { AvatarGroupPropTypes };
