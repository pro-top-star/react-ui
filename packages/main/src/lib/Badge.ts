import { Badge } from '../webComponents/Badge';
import type { BadgePropTypes } from '../webComponents/Badge';

export { Badge };
export type { BadgePropTypes };
