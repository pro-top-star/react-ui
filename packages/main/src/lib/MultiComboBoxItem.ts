import { MultiComboBoxItem } from '../webComponents/MultiComboBoxItem';

export { MultiComboBoxItem };
