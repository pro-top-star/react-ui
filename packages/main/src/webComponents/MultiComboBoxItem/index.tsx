import { withWebComponent } from '@ui5/webcomponents-react/lib/withWebComponent';
import '@ui5/webcomponents/dist/MultiComboBoxItem';
import React, { FC } from 'react';
import { WithWebComponentPropTypes } from '../../internal/withWebComponent';

export interface MultiComboBoxItemPropTypes extends WithWebComponentPropTypes {
  /**
   * Defines the selected state of the <code>ui5-mcb-item</code>.
   */
  selected?: boolean;
  /**
   * Defines the text of the <code>ui5-cb-item</code>.
   */
  text?: string;
}

/**
 * <code>import { MultiComboBoxItem } from '@ui5/webcomponents-react/lib/MultiComboBoxItem';</code>
 * <br />
 * <a href="https://sap.github.io/ui5-webcomponents/playground/components/MultiComboBoxItem" target="_blank">UI5 Web Components Playground</a>
 */
const MultiComboBoxItem: FC<MultiComboBoxItemPropTypes> = withWebComponent<MultiComboBoxItemPropTypes>(
  'ui5-mcb-item',
  ['text'],
  ['selected'],
  [],
  []
);

MultiComboBoxItem.displayName = 'MultiComboBoxItem';

MultiComboBoxItem.defaultProps = {
  selected: false
};

export { MultiComboBoxItem };
