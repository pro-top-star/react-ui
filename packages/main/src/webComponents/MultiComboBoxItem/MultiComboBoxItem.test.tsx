import { MultiComboBoxItem } from '@ui5/webcomponents-react/lib/MultiComboBoxItem';
import { mount } from 'enzyme';
import React from 'react';

describe('MultiComboBoxItem', () => {
  test('Basic Test (generated)', () => {
    const wrapper = mount(<MultiComboBoxItem />);
    expect(wrapper.render()).toMatchSnapshot();
  });
});
