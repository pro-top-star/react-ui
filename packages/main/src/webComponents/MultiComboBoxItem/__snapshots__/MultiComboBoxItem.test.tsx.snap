// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MultiComboBoxItem Basic Test (generated) 1`] = `
<ui5-mcb-item
  text=""
/>
`;
